bfdd/bfd.c

```diff
@@ -1226,27 +1226,6 @@ void bs_final_handler(struct bfd_session *bs)
 	/* Apply new transmission timer immediately. */
 	ptm_bfd_start_xmt_timer(bs, false);
 
-	/*
-	 * Detection timeout calculation:
-	 * The minimum detection timeout is the remote detection
-	 * multipler (number of packets to be missed) times the agreed
-	 * transmission interval.
-	 *
-	 * RFC 5880, Section 6.8.4.
-	 *
-	 * TODO: support sending/counting more packets inside detection
-	 * timeout.
-	 */
```

Comment:
was sending/counting more packets related to your fix, or is there something more in the RFC that tells, we should count for received / emitted packets ?

Reply:
This TODO comment is weird and not related to detection timer update, so I removed it.

Reply:
@pguibert6WIND do you have any other questions?

Reply:
Actually, the TODO should be kept. this was for accounting packets.
This can be also  used to calculate more accurately timeout on slicing windows..

Reply:
If you understand what this TODO means and how it is related to the detection timeout, please, explain this to me because I don't understand it at all.
Otherwise I don't see the reason to keep this TODO.

Reply:
@pguibert6WIND please, update.

Reply:
This TODO I've made during a refactory a while ago... The intention was to have a more fine grained status/statistic for BFD session: e.g. how many packets were received/sent during the sliding window for detection interval (time * detection timer).

I think this can be removed since it is not a requirement for anything, but a nice feature (e.g. see jitter/lost packets). If anyone wants to implement it they don't need the TODO comment.